### PR TITLE
complete micromasters summary report

### PR DESCRIPTION
**Pull request by abeglova**
# What are the relevant tickets?
closes https://github.com/mitodl/ol-data-platform/issues/528

# Description (What does it do?)
This pr adds the remaining missing fields to marts__micromasters_summary. Specifically
    verified_enrollments
    unique_verified_users
    course_certificates
    unique_course_certificate_earners
    program_certificates

It also gets rid of duplicated logic between 
int__mitx__courserun_enrollments and int__micromasters__course_enrollments. int__micromasters__course_enrollments now just gets data from int__mitx__courserun_enrollments

### CODE REVIEW

File: src/ol_dbt/models/marts/micromasters/marts__micromasters_summary.sql [hunk 1]
```diff
@@ -3,36 +3,137 @@ with enrollments as (
...
         , count(*) as total_enrollments
         , count(distinct user_email) as unique_users
-        , count(distinct user_country) as unique_countries
+        , count(distinct user_address_country) as unique_countries
+        , count_if(courserunenrollment_enrollment_mode = 'verified') as verified_enrollments
+        , count(distinct case when courserunenrollment_enrollment_mode = 'verified' then user_email end)
+            as unique_verified_users
     from enrollments
-    group by program_title
+    group by program_title, micromasters_program_id
```
Comment (rachellougee) [inline src/ol_dbt/models/marts/micromasters/marts__micromasters_summary.sql:27]:
```suggestion
    group by micromasters_program_id
```

File: src/ol_dbt/models/marts/micromasters/_marts_micromasters__models.yml [hunk 2]
```diff
@@ -10,19 +10,45 @@ models:
...
+    description: int, count of unique countries of enrolled users
+    tests:
+    - not_null
+  - name: verified_enrollments
+    description: int, count of verified (paid) enrollments in the program's courses.
+    tests:
+    - not_null
+  - name: unique_verified_users
+    description: int, count of unique verified users enrolled in the program's courses.
+      Email is used to dedupe users accross mitxonline and edxorg so this number might
```
Comment (rachellougee) [inline src/ol_dbt/models/marts/micromasters/_marts_micromasters__models.yml:34]: To be clear, it's not that email is used to dedupe users, it's that we use their linked accounts from MicroMasters social auth to dedupe. The identified duplicate users could have the same email or two different emails across MITx Online and edX.org. As long as their accounts are linked on MM, they are identified as the same user regardless of email.

**Review by rachellougee — APPROVED** [review]
LGTM. Just two comments, not blocker

File: src/ol_dbt/models/marts/micromasters/_marts_micromasters__models.yml (hunk 2 above)
Comment (abeglova) [reply src/ol_dbt/models/marts/micromasters/_marts_micromasters__models.yml:34]: That should be close enough, since this data is just used for marketing

**Review by rachellougee — APPROVED** [review]
LGTM

**Pull request closed without merging**